server/routes/ticket.js

```diff
@@ -1,29 +1,20 @@
 import express from 'express';
 import validate from 'express-validation';
 import paramValidation from './validation/ticket';
-
 import ticketController from '../controllers/ticket';
 
 const router = express.Router();  // eslint-disable-line new-cap
 
 router.route('/all')
-
   .get(validate(paramValidation.getTicketsByDate), ticketController.getTicketsByDate);
 
 router.route('/:id')
-
-  /** GET /api/v1/ticket/:id - Get a ticket by Id */
   .get(validate(paramValidation.byId), ticketController.getById)
-
-  /** PUT /api/v1/ticket/:id - Update a ticket */
+  .post(validate(paramValidation.byId), ticketController.close)
```

Comment:
That makes enough sense. In that case, though, I think it'd make more sense to have it be a `PUT`, since `POST` is generally for creation rather than updating (which is essentially what we're doing when closing a ticket). But I see that `PUT` already exists for this route, so I guess `POST` is the only option.